### PR TITLE
adding commandd.

### PR DESCRIPTION
**Pull request by awais786**
Some experiment.

### CODE REVIEW

**Comment by robrap** [comment]
@awais786: _If this goes beyond an experiment_, can you add an explanation and also explain why this should live in this repo. No need to respond unless you push forward with this later. Thank you!

**Comment by awais786** [comment]
> @awais786: _If this goes beyond an experiment_, can you add an explanation and also explain why this should live in this repo. No need to respond unless you push forward with this later. Thank you!

**Comment by awais786** [comment]
@robrap I am checking how management command works from this package. But missing some thing. Package is installed in shell but not command not runs successfully.

**Comment by regisb** [comment]
@awais786  I think you should move the `manage_users_by.py` file from the `admin/` to the `management/commands` folder.

**Comment by awais786** [comment]
> @awais786 I think you should move the `manage_users_by.py` file from the `admin/` to the `management/commands` folder.

@regisb I am checking some thing. But If i continue with this I will move that there. Thanks.

File: edx_django_utils/admin/manage_users_by.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+from django.contrib.auth.hashers import identify_hasher, is_password_usable
+from django.contrib.auth.models import Group
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from django.utils.translation import gettext as _
+
+logger = getLogger(__name__)  # pylint: disable=invalid-name
+
+
+try:
```
Comment (adzuci) [inline edx_django_utils/admin/manage_users_by.py:18]: Hey @awais786, thanks for putting this PR up!  I and Saad agree that the try catch statement is easier than writing a `post_save` signal handler, though it's not clear to me what the pros and cons are.

Do you think we can use the same technique for the generate password logic or somehow have an if statement that only hits that logic if in an edxapp app?

https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/management/commands/manage_user.py#L100-L110

e.g. 

```
try:
    from openedx.core.djangoapps.user_authn.utils import generate_password
except ImportError:
    UserProfile = None
```

Comment (awais786) [reply edx_django_utils/admin/manage_users_by.py:18]: @adzuci  Sorry I missed this msg. 
Yes you can do this as well. I don't see any issue. Its already implemented at several places. Please check this link.
https://github.com/edx/edx-enterprise/blob/3215aa03dfcee02cbb6d795fb29e025b4656398a/enterprise/utils.py#L45

**Pull request closed without merging**